.github/workflows/publish-master-merges.yaml

```diff
@@ -17,19 +17,28 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - uses: WyriHaximus/github-action-get-previous-tag@v1
-        id: get-latest-tag
-      - id: get-stable-version
-        run: echo "stable_version=$(echo ${{ steps.get-latest-tag.outputs.tag }} | cut -d'.' -f1-3)" >> $GITHUB_ENV
+      - uses: azure/setup-helm@v1
+        with:
+          version: '3.8.0'
+        id: setup-helm
+      - id: get-latest-helm-charts
+        run: |
+          helm repo add renku https://swissdatasciencecenter.github.io/helm-charts/
+          helm repo update
+          helm search repo renku --devel -o yaml > .latest-charts.yaml
+      - id: get-latest-version
+        uses: mikefarah/yq@v4.19.1
```

Comment:
I wonder if `jq` is present in the actions builders already by default - if that's the case I would skip this extra dependency and output to json in the step above.

Reply:
Ok I will test for that.

Reply:
yeah that works, see here: https://github.com/SwissDataScienceCenter/renku/runs/5093411722?check_suite_focus=true

One sec let me just remove the stuff I added to test

Reply:
looks good, thanks!